CodeGeneration/Sources/generate-swiftbasicformat/BasicFormatFile.swift

```diff
@@ -122,6 +122,19 @@ let basicFormatFile = SourceFile {
     }
 
     FunctionDecl("open func requiresLeadingSpace(_ token: TokenSyntax) -> Bool") {
+      SwitchStmt("""
+        switch (token.tokenKind, token.previousToken(viewMode: .sourceAccurate)?.tokenKind) {
```

Comment:
What do you think about flipping the order of the elements in this tuple here so we are matching agains the token kinds in the same order as they appear in source code?